#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-O_DIRECT.dpatch by  <sgran@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Need O_DIRECT for the --direct switch to work

@DPATCH@
diff -urNad hdparm-9.12~/hdparm.c hdparm-9.12/hdparm.c
--- hdparm-9.12~/hdparm.c	2009-02-26 18:10:22.000000000 +0000
+++ hdparm-9.12/hdparm.c	2009-03-14 20:40:53.000000000 +0000
@@ -4,6 +4,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
+#define __USE_GNU      /* for O_DIRECT */
 #include <fcntl.h>
 #include <errno.h>
 #include <ctype.h>
